test/system/clients_test.rb

```diff
@@ -118,11 +118,14 @@ class ClientsTest < ApplicationSystemTestCase
     refute page.has_text? 'Fluent'
   end
 
-  test 'superadmin can delete client' do
+  test 'superadmin_can_delete_client' do
```

Comment:
Accidental change? :)

Reply:
nope, not really. Snake case test names are easier to call as single run by `rails test -n /superadmin_can_delete_client/ test/system/clients_test.rb`. This way it can be copy and pasted. Doing `rails test -n '/superadmin can delete client/' test/system/clients_test.rb` did not work here usually.
For unknown reason I was not able to run our system tests by line as documented by rails.

Reply:
nope, not really. Snake case test names are easier to call as single run by rails test -n /superadmin_can_delete_client/ test/system/clients_test.rb. This way it can be copy and pasted. Doing rails test -n '/superadmin can delete client/' test/system/clients_test.rb did not work here usually.
For unknown reason I was not able to run our system tests by line as documented by rails.

Reply:
Maybe you know why I canot run `rails test test/system/clients_test.rb:122`.
Here I still get all the tests in the file executed, if I do that

Reply:
This test now starts at line 121 not 122. (Although locally both commands run successfully. )

Reply:
that should not matter. It runs only one single test from the file on your box?

Reply:
Hmm both commands run all tests as well here, even though [passing a line number seems to be supported](http://guides.rubyonrails.org/testing.html#the-rails-test-runner)

Same with model tests, so it's not related to system tests either.